models/sudoswap/ethereum/sudoswap_ethereum_schema.yml

```diff
@@ -222,4 +222,55 @@ models:
       - *block_number
       - *tx_from
       - *tx_to
-      - *unique_trade_id
\ No newline at end of file
+      - *unique_trade_id
+
+  - name: sudoswap_ethereum_pools
+    meta:
+      blockchain: ethereum
+      project: sudoswap
+      contributors: [niftytable]
+    config:
+      tags: ['ethereum','sudoswap','pools']
+    description: >
+        sudoswap pools on Ethereum
```

Comment:
now that you've defined your unique key in the model config, it's good practice to apply a test to this table in the schema file on those columns. please see for example:
https://github.com/duneanalytics/spellbook/pull/1472/files#diff-7da167e3f4a73d5c5c2e3d245dc8c73c88619f49d08d0fea792a4bd8c54382aeR14

Reply:
Ok, adding that dbt_utils.unique_combination_of_column test